packages/flutter_tools/lib/src/fuchsia/cmx_validator.dart

```diff
@@ -0,0 +1,17 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+
+import '../base/common.dart';
+import '../base/file_system.dart';
+import '../project.dart';
+
+Future<void> validateCmxFile(FuchsiaProject fuchsiaProject) async {
```

Comment:
The only callsites of this function call it right before calling `buildFuchsia`. I'd suggest moving this function to the same library as `buildFuchsia` and making it library-private, that is `_validateCmxFile`, and just call it at the top of `buildFuchsia`.

Reply:
Good call, done.